backend/src/main/java/ai/giskard/service/TransformationFunctionService.java

```diff
@@ -29,7 +30,13 @@ protected TransformationFunction create(TransformationFunctionDTO dto) {
         if (function.getArgs() != null) {
             function.getArgs().forEach(arg -> arg.setFunction(function));
         }
-        function.setVersion(transformationFunctionRepository.countByNameAndModule(function.getName(), function.getModule()) + 1);
+
+        if (Strings.isBlank(function.getDisplayName())) {
```

Comment:
@kevinmessiaen  it's a pity to duplicate this part 3 times, can you refactor it to share between the 3 function types?

for example by extracting a common method like
```
    @NotNull
    public static <T extends Callable> T intializeFunction(T function, CallableRepository repo) {
        if (function.getArgs() != null) {
            function.getArgs().forEach(arg -> arg.setFunction(function));
        }

        if (Strings.isBlank(function.getDisplayName())) {
            function.setDisplayName(function.getModule() + "." + function.getName());
        }

        function.setVersion(repo.countByDisplayName(function.getDisplayName()) + 1);

        return function;
    }
```